### ISSUE TITLE
setBottomPadding error?

### ISSUE THREAD

**Issue by davidseek** [stewwan/capacitor-intercom#6]
Hey, while implementing the new version I'm edging on this:

`⚡️  Error: Plugin IntercomPlugin does not respond to method call "setBottomPadding" using selector "setBottomPadding:".  `

Any idea what that's about? 

Implementation pretty straight forward:

```
const paddingOptions = {
   value: '128'
}

intercom.setBottomPadding(paddingOptions)
   .then(() => {

      console.log('Intercom padding success')
    })
    .catch(error => {

      console.log('Intercom padding error: ', error)
})
```

**Comment by davidseek** [comment]
I have double checked, that I ran `cap copy` and done. 
Still the same error.

**Comment by davidseek** [comment]
I assume I found the problem in the lib:

<img width="1129" alt="Screen Shot 2019-10-13 at 5 09 04 PM" src="https://user-images.githubusercontent.com/17073950/66724564-350ad080-eddc-11e9-84ff-9fccee12c1ba.png">

**Comment by davidseek** [comment]
Pull request: 

https://github.com/stewwan/capacitor-intercom/pull/7

**Comment by stewwan** [comment]
yeah I didn't test. thanks for the PR

**Comment by stewwan** [comment]
patched in 0.2.1

**Comment by davidseek** [comment]
For Android -> Pull request: https://github.com/stewwan/capacitor-intercom/pull/8

**Comment by stewwan** [comment]
Thanks! released 0.2.2
